### ISSUE TITLE
Adding a new event fails

### ISSUE THREAD

**Issue by jackahl** [plone/volto#612]
@tisto At the moment it is not possible to add new events with Volto. Tying to do that sends you to an empty page.

**Comment by tisto** [comment]
Steps to reproduce:

1) Go to https://volto.kitconcept.com
2) Login as admin:admin
3) Add new event: https://volto.kitconcept.com/add?type=Event
-> Empty page
-> Open network tab, request to https://volto.kitconcept.com/api/@types/Event fails with a 500 error. Trace:

```
{
  "message": "No converter for making <function default_timezone at 0x7f11e46a2b18> (<type 'function'>) JSON compatible.", 
  "traceback": [
    "File \"/plone/lib/python2.7/site-packages/ZPublisher/Publish.py\", line 138, in publish", 
    "    request, bind=1)", 
    "", 
    "  File \"/plone/lib/python2.7/site-packages/ZPublisher/mapply.py\", line 77, in mapply", 
    "    if debug is not None: return debug(object,args,context)", 
    "", 
    "  File \"/plone/lib/python2.7/site-packages/ZPublisher/Publish.py\", line 48, in call_object", 
    "    result=apply(object,args) # Type s<cr> to step into published object.", 
    "", 
    "  File \"/plone/lib/python2.7/site-packages/plone/rest/service.py\", line 23, in __call__", 
    "    return self.render()", 
    "", 
    "  File \"/plone/lib/python2.7/site-packages/plone/restapi/services/__init__.py\", line 19, in render", 
    "    content = self.reply()", 
    "", 
    "  File \"/plone/lib/python2.7/site-packages/plone/restapi/services/types/get.py\", line 52, in reply", 
    "    self.request", 
    "", 
    "  File \"/plone/lib/python2.7/site-packages/plone/restapi/types/utils.py\", line 208, in get_jsonschema_for_portal_type", 
    "    fti, context, request, excluded_fields=excluded_fields)", 
    "", 
    "  File \"/plone/lib/python2.7/site-packages/plone/restapi/types/utils.py\", line 194, in get_jsonschema_for_fti", 
    "    'properties': json_compatible(properties),", 
    "", 
    "  File \"/plone/lib/python2.7/site-packages/plone/restapi/serializer/converters.py\", line 72, in json_compatible", 
    "    return IJsonCompatible(value, None)", 
    "", 
    "  File \"/plone/lib/python2.7/site-packages/zope/component/hookable.py\", line 33, in __call__", 
    "    return self.__implementation(*args, **kw)", 
    "", 
    "  File \"/plone/lib/python2.7/site-packages/zope/component/hooks.py\", line 119, in adapter_hook", 
    "    return siteinfo.adapter_hook(interface, object, name, default)", 
    "", 
    "  File \"/plone/lib/python2.7/site-packages/plone/restapi/serializer/converters.py\", line 133, in dict_converter", 
    "    values = list(map(json_compatible, values))", 
    "", 
    "  File \"/plone/lib/python2.7/site-packages/plone/restapi/serializer/converters.py\", line 72, in json_compatible", 
    "    return IJsonCompatible(value, None)", 
    "", 
    "  File \"/plone/lib/python2.7/site-packages/zope/component/hookable.py\", line 33, in __call__", 
    "    return self.__implementation(*args, **kw)", 
    "", 
    "  File \"/plone/lib/python2.7/site-packages/zope/component/hooks.py\", line 119, in adapter_hook", 
    "    return siteinfo.adapter_hook(interface, object, name, default)", 
    "", 
    "  File \"/plone/lib/python2.7/site-packages/plone/restapi/serializer/converters.py\", line 133, in dict_converter", 
    "    values = list(map(json_compatible, values))", 
    "", 
    "  File \"/plone/lib/python2.7/site-packages/plone/restapi/serializer/converters.py\", line 72, in json_compatible", 
    "    return IJsonCompatible(value, None)", 
    "", 
    "  File \"/plone/lib/python2.7/site-packages/zope/component/hookable.py\", line 33, in __call__", 
    "    return self.__implementation(*args, **kw)", 
    "", 
    "  File \"/plone/lib/python2.7/site-packages/zope/component/hooks.py\", line 119, in adapter_hook", 
    "    return siteinfo.adapter_hook(interface, object, name, default)", 
    "", 
    "  File \"/plone/lib/python2.7/site-packages/plone/restapi/serializer/converters.py\", line 86, in default_converter", 
    "    ' {0!r} ({1}) JSON compatible.'.format(value, type(value)))"
  ], 
  "type": "TypeError"
}
```

**Comment by davisagli** [comment]
@tisto what version of plone.restapi?

**Comment by sneridagh** [comment]
@tisto It's fixed since 3.5.2, now it's working fine, thanks to @davisagli :)

However, it seems that editing the object fail to get the current value in the form, I will open an issue for that.
